agasc/tests/test_agasc_2.py

```diff
@@ -581,3 +581,44 @@ def test_get_supplement_table_obs():
 def test_get_supplement_table_obs_dict():
     obs = agasc.get_supplement_table("obs", as_dict=True)
     assert isinstance(obs, dict)
+
+
+def test_write():
+    import tempfile
+    from pathlib import Path
+
+    import tables
+
+    from agasc import write_agasc
+
+    with (
+        tempfile.TemporaryDirectory() as tempdir,
```

Comment:
Use the pytest `tmp_path` fixture. https://docs.pytest.org/en/7.1.x/how-to/tmp_path.html

Reply:
Done